### PR TITLE
feat(canary): export web3modal Canary image

### PR DESCRIPTION
**Pull request by arein**
Very simple Playwright test to get full end-2-end confidence/latency metrics including:
1. Explorer API
2. Blockchain API
3. Other tools

# Testing

Built successfully into ECR and tested locally
https://github.com/WalletConnect/web3modal/actions/runs/7835190121/job/21380181938

# Changes

- [feat: export web3modal Canary image](https://github.com/WalletConnect/web3modal/commit/ae1cf0b7c9c8a8bc57a4897dd538d1b082c68ca0)

# Associated Issues

closes #...

### CODE REVIEW

File: apps/laboratory/tests/canary.spec.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  await modalValidator.expectDisconnected()
+  await walletValidator.expectDisconnected()
+})
+
+testMW(
+  'it should sign',
+  async ({ modalPage, walletPage, modalValidator, walletValidator, browserName }) => {
+    // Canary doesn't need all platforms
+    if (browserName !== 'chromium' || modalPage.library !== 'ethers') {
+      testMW.skip()
```
Comment (tomiir) [inline apps/laboratory/tests/canary.spec.ts:33]: Why skip here and empty return on the others?

Comment (arein) [reply apps/laboratory/tests/canary.spec.ts:33]: It's an intricacy of Playwright

File: apps/laboratory/tests/canary.spec.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    await modalValidator.expectConnected()
+    await walletValidator.expectConnected()
+  }
+)
+
+testMW.afterEach(async ({ modalPage, modalValidator, walletValidator, browserName }) => {
+  // Canary doesn't need all platforms
+  if (browserName !== 'chromium' || modalPage.library !== 'ethers') {
+    return
+  }
```
Comment (tomiir) [inline apps/laboratory/tests/canary.spec.ts:22]: could be abstracted under  your 3 time no-repeat rule but not a big deal

**Review by tomiir — APPROVED** [review]
LGTM!

**Pull request merged**